app/views/proposals/_archive_link.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="col-sm-2">
-  <%- unless @approved_data.rows.size < @CLOSED_PROPOSAL_LIMIT %>
-    <%= link_to "View the archive", url_for(:controller => 'proposals', :action => 'archive') %>
-  <%- end %>
+  <% if @approved_data.rows.size > @closed_proposal_limit %>
```

Comment:
oh interesting, I assume it was not intentional. Do you remember why we did that?